packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -0,0 +1,159 @@
+# Accordion
+
+An accordion is used to allow users to toggle between different content sections in a compact vertical stack.
+
+This document outlines the API of `Accordion` component.
+
+<div align="center">
+  <img src="./accordion-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Accordion](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-619941&t=Rcrabprdzrz7JdId-0)
+
+## Interactions
+
+## API
+
+Annotated component with props ([figma](https://www.figma.com/file/wzCTQ9kMjWWHKFu7kCzYGi/Accordion-breakdown?type=design&node-id=0-1&t=GfrkTz89y2M46wQN-0)):
+
+![Accordion breakdown](./accordion-breakdown.png)
+
+Sample usage, composes `Accordion` and `AccordionItem`:
+
+```jsx
+import { Accordion, AccordionItem } from '@razorpay/blade';
+
+<Accordion variant="ordered" defaultIndex={0}>
+  <AccordionItem
+    title="Can I create linked accounts using Route?"
+    description="You can use Razorpay Route from the Dashboard or using APIs to transfer money to customers."
+  >
+    <div>Slot</div>
+  </AccordionItem>
+  <AccordionItem title="How can I transfer money to customers?" description="Just use Razorpay" />
+</Accordion>;
+```
+
+### Accordion
+
+| Prop         | Type                   | Default     | Description                                                    | Required |
+| ------------ | ---------------------- | ----------- | -------------------------------------------------------------- | -------- |
+| variant      | `unordered`, `ordered` | `unordered` | `ordered` shows a numeric prefix to titles                     |          |
```

Comment:
Why do we need `unordered` here? I can only see number & icon on Figma. 

Also, do we need to call it ordered or do we call it `numeric`? or even `showNumberPrefix={true}`

Reply:
+1

We shall keep it inline with design. `leading?: number` even icon will be directly applied on Item and we shall then also have a restriction when someone has passed leading value and then still passing icon in the accordion so both shouldn't be allowed at the same time

Reply:
Made it `ordered` to keep it similar to `List` component (because we want the nos to be auto added in accordion items). Alternatively can add a boolean for controlling to the root accordion.

Reply:
Added in https://github.com/razorpay/blade/pull/1189/commits/f6a8627de1aa246be547492accb9f7052dc8381e